html/mycakeapp/src/Controller/AuctionController.php

```diff
@@ -56,52 +53,74 @@ public function view($id = null)
 			$bidinfo->biditem_id = $id;
 			// 最高金額のBidrequestを検索
 			$bidrequest = $this->Bidrequests->find('all', [
-				'conditions'=>['biditem_id'=>$id], 
+				'conditions' => ['biditem_id' => $id],
 				'contain' => ['Users'],
-				'order'=>['price'=>'desc']])->first();
+				'order' => ['price' => 'desc']
+			])->first();
 			// Bidrequestが得られた時の処理
-			if (!empty($bidrequest)){
+			if (!empty($bidrequest)) {
 				// Bidinfoの各種プロパティを設定して保存する
 				$bidinfo->user_id = $bidrequest->user->id;
 				$bidinfo->user = $bidrequest->user;
 				$bidinfo->price = $bidrequest->price;
 				$this->Bidinfo->save($bidinfo);
 			}
 			// Biditemのbidinfoに$bidinfoを設定
-			$biditem->bidinfo = $bidinfo;		
+			$biditem->bidinfo = $bidinfo;
 		}
 		// Bidrequestsからbiditem_idが$idのものを取得
 		$bidrequests = $this->Bidrequests->find('all', [
-			'conditions'=>['biditem_id'=>$id], 
+			'conditions' => ['biditem_id' => $id],
 			'contain' => ['Users'],
-			'order'=>['price'=>'desc']])->toArray();
+			'order' => ['price' => 'desc']
+		])->toArray();
 		// オブジェクト類をテンプレート用に設定
 		$this->set(compact('biditem', 'bidrequests'));
 	}
-
 	// 出品する処理
 	public function add()
 	{
 		// Biditemインスタンスを用意
 		$biditem = $this->Biditems->newEntity();
 		// POST送信時の処理
 		if ($this->request->is('post')) {
-			// $biditemにフォームの送信内容を反映
-			$biditem = $this->Biditems->patchEntity($biditem, $this->request->getData());
-			// $biditemを保存する
-			if ($this->Biditems->save($biditem)) {
-				// 成功時のメッセージ
-				$this->Flash->success(__('保存しました。'));
-				// トップページ（index）に移動
-				return $this->redirect(['action' => 'index']);
+			$tmp_data = $this->request->getData();
+
+			// 本当に画像なのかを判断(gif,jpeg,png形式のみ許可)
+			$image_check = exif_imagetype($tmp_data['image_path']['tmp_name']);
+			if ($image_check === 1 || 2 || 3) {
+				//アップロードされた画像名に日時とユーザーidを加える（画像名重複防止）
+				$file_name =   date("YmdHis") .  $tmp_data['user_id'] . $tmp_data['image_path']['name'];
+				//画像保存先パス
+				$img_save_path = WWW_ROOT . 'img/' . $file_name;
+				//画像の保存処理
+				move_uploaded_file($tmp_data['image_path']['tmp_name'], $img_save_path);
+				$tmp_data['image_path'] = $file_name;
+			} else {
+				$this->Flash->error(__('画像をアップロードして下さい'));
+			}
+			if ($tmp_data['image_path'] = $file_name) {
```

Comment:
条件式が代入文になっておりますが、もしかしてこう書く予定でしょうか
```suggestion
			if ($tmp_data['image_path'] === $file_name) {
```
であれば提出前に動作確認をお願いします！

Reply:
完全なチェック漏れで申し訳ございません。
8c2bf22 こちらのコミットで修正しました（コミットコメントにこの件入れ忘れてしまいました…）